packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -668,6 +675,13 @@ export class ListItem
     const showBorder = selectionMode !== "none" && selectionAppearance === "border";
     const borderSelected = showBorder && selected;
     const borderUnselected = showBorder && !selected;
+    const containerNotInteractive =
```

Comment:
I _think_ this can be simplified to:

```tsx
const containerInteractive =
  interactionMode === "interactive" ||
  (
    interactionMode === "static" &&
    selectionMode !== "none" &&
    selectionAppearance === "border"
  );

// ...

[CSS.containerHover]: containerInteractive,
```

Reply:
Thanks for catching that, this update works!